app.py

```diff
@@ -33,8 +34,12 @@ def index():
     """Application landing page."""
     SITE_ROOT = os.path.realpath(os.path.dirname(__file__))
     json_url = os.path.join(SITE_ROOT, 'static', 'data.json')
+    config_path = os.path.dirname(os.path.abspath(__file__)) + '/config.yaml'
```

Comment:
Can't we use SITE_ROOT again?  (After making sure the variable is set to an absolute path.)